packages/core/test-integrations/test-execution-result-fixtures.ts

```diff
@@ -0,0 +1,83 @@
+import { assert } from "chai";
```

Comment:
This test ensures that the fixture that we use in the `test/` folder is correct. It actually runs every call that the fixture is meant to simulate.

This allows us to keep `test/` independent from Hardhat network and fast, but be confident it is correct.